hg-fast-export.py

```diff
@@ -149,11 +149,14 @@ def sanitize_name(name,what="branch"):
   """Sanitize input roughly according to git-check-ref-format(1)"""
 
   def dot(name):
```

Comment:
From the commit message it is unclear what you are trying to achieve, anyway this is not a good idea, check the newly added comment in 46bf316a3cca059d60c0701009f1fe1c6d6b51f5 for the explanation.